Experiments/Experiments/FeatureFlag.swift

```diff
@@ -86,6 +86,10 @@ public enum FeatureFlag: Int {
     ///
     case productsOnboarding
 
+    /// Performs XMLRPC only when user opts for signing in using .org site credentials
+    ///
+    case checkXMLRPCOnlyIfSigningInUsingSiteCredentials
```

Comment:
Super nit: this can be simplified
```suggestion
    case checkXMLRPCOnlyForSiteCredentialLogin
```